src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -212,8 +214,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("TypedDataCollection", bool.TrueString);
             response.Capabilities.Add("WorkerStatus", bool.TrueString);
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
-            response.Capabilities.Add("HandlesInvocationCancelMessage", bool.TrueString);
-
+            response.Capabilities.Add("EnableUserCodeException", bool.TrueString);  //enableUserCodeExceptionOption.ToString());
```

Comment:
This is not using the parameter. Remove comment and pass the actual parameter value.